### PR TITLE
fix typo and add .idea folder in gitignore

### PR DESCRIPTION
**Pull request by Ivanshamir**
Here I fix a typo and add .idea folder in gitignore file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.idea/
+
```
Comment (clue) [inline .gitignore:2]: This should probably be part of your global gitignore: https://stackoverflow.com/questions/7335420/global-git-ignore

**Review by clue — COMMENTED** [review]
@Ivanshamir Thanks for spotting and filing this PR, the fix LGTM!

It looks like the changes in the `.gitignore` are unrelated, may I ask you to revert (amend) these changes?

File: .gitignore (hunk 1 above)
Comment (Ivanshamir) [reply .gitignore]: @clue Ok. I have reverted the gitignore code. Can you please recheck?

Comment (clue) [reply .gitignore]: Looks like your last commit introduced some additional changes? Can you revert those and squash your changes into a single commit?

**Pull request closed without merging**